source/FreeRTOS_DNS_Cache.c

```diff
@@ -624,24 +664,33 @@
                     for( xSubEntry = 0; xSubEntry < pxRow->ucNumIPAddresses; xSubEntry++ )
                     {
                         char pcAddress[ 40 ] = "";
-                        #if ( ipconfigUSE_IPv6 != 0 )
 
+                        switch( pxRow->xAddresses[ 0 ].xIs_IPv6 )
```

Comment:
Are we making sure these values are initialized properly? Otherwise it might cause issue.

Reply:
All these changes are done and tested as per the assumption that the previous version of this code is working fine. In this case this changes is made analogous to: `if( pxRow->xAddresses[ 0 ].xIs_IPv6 != pdFALSE )`  in the previous version.